tools/castle-editor/code/framedesign.pas

```diff
@@ -2144,6 +2254,7 @@ procedure TDesignFrame.MenuItemViewportCamera2DViewInitialClick(
   V := SelectedViewport;
   V.Setup2D;
   ModifiedOutsideObjectInspector;
+  RecordUndo('MenuItemViewportCamera2DViewInitialClick'); //whatever that means?
```

Comment:
For this and following `//whatever that means?`, the relevant menu item captions (maybe even shorter) are mostly OK. This one should be

```
RecordUndo('Camera Setup for 2D View and Projection');
```

Reply:
Thanks, fixed!